config/scenario_config_21_EU11_Fit_for_55_sensitivity.csv

```diff
@@ -1,6 +1,6 @@
 title;start;cm_bioprod_regi_lim;cm_BioSupply_Adjust_EU;cm_import_tax;cm_bioenergy_SustTax;c_regi_earlyreti_rate;cm_implicitQttyTarget;cm_share_LDV_sales;cm_adj_coeff_multiplier;cm_adj_seed_multiplier;cm_vehiclesSubsidies;cm_adj_coeff;cm_adj_seed;cm_incolearn;cm_inco0Factor;cm_learnRate;cm_storageFactor;cm_renewables_floor_cost;cm_indst_costDecayStart;cm_indst_H2costDecayEnd;cm_build_H2costDecayStart;cm_build_H2costDecayEnd;cm_flex_tax;cm_shSynLiq;cm_shSynGas;cm_CCS_markup;cm_Industry_CCS_markup;cm_CCSRegiPol;c_ccsinjecratescen;c_budgetCO2from2020;CES_parameters;cm_calibration_string;cm_calibration_FE;cm_wind_offshore;cm_iteration_max;regionmapping;extramappings_historic;cm_rcp_scen;cm_iterative_target_adj;subsidizeLearning;carbonprice;cm_peakBudgYr;cm_taxCO2inc_after_peakBudgYr;cm_CO2priceRegConvEndYr;cm_emiscen;cm_fetaxscen;cm_co2_tax_growth;techpol;cm_multigasscen;regipol;cm_emiMktTarget;cm_emiMktTargetDelay;cm_NucRegiPol;cm_CoalRegiPol;cm_proNucRegiPol;cm_altFeEmiFac;cm_co2_tax_2020;optimization;c_testOneRegi_region;cm_nash_mode;buildings;transport;industry;cm_EDGEtr_scen;CCU;water;cm_prtpScen;cm_POPscen;cm_GDPscen;cm_demScen;cm_startyear;path_gdx;path_gdx_ref
 # _____Calibration_cplx_____;0;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-xx_Calib_FE_med;0;off;3;EUR_regi.pebiolc 1.0;1.5;GLO 0.09, EUR_regi 0.15;off;off;off;off;off;off;off;off;apcardiEfft 0.8, apcardiEffH2t 0.675;apcarelt 0.1;off;off;0.05;0.1;0.05;0.1;0;0;0;off;off;off;1;0;calibrate;FE_med;medium;1;1;./config/regionmapping_21_EU11.csv;./config/extramapping_EU27.csv;none;0;off;none;2100;3;2050;1;3;1.05;none;2;regiCarbonPrice;off;0;off;off;off;EUR_regi, NEU_regi;-1;nash;DEU;parallel;simple;complex;fixed_shares;Mix1;off;heat;3;pop_SSP2EU;gdp_SSP2EU;gdp_SSP2EU;2005;;
+#  xx_Calib_FE_med (needs adjustment regarding cm_calibration_FE);0;off;3;EUR_regi.pebiolc 1.0;1.5;GLO 0.09, EUR_regi 0.15;off;off;off;off;off;off;off;off;apcardiEfft 0.8, apcardiEffH2t 0.675;apcarelt 0.1;off;off;0.05;0.1;0.05;0.1;0;0;0;off;off;off;1;0;calibrate;FE_med;medium;1;1;./config/regionmapping_21_EU11.csv;./config/extramapping_EU27.csv;none;0;off;none;2100;3;2050;1;3;1.05;none;2;regiCarbonPrice;off;0;off;off;off;EUR_regi, NEU_regi;-1;nash;DEU;parallel;simple;complex;fixed_shares;Mix1;off;heat;3;pop_SSP2EU;gdp_SSP2EU;gdp_SSP2EU;2005;;
```

Comment:
Scenarios that use transport complex can be removed from this scenario config, removing all scenarios dependable of the deprecated `cm_calibration_FE` from the scenario config as consequence.
This can be included in this PR or I could add this change afterwards. Whatever you prefer.

Reply:
I think, the whole config file needs a review, so please do it afterwards